ledger/sandbox/src/main/scala/com/digitalasset/platform/store/Conversions.scala

```diff
@@ -7,185 +7,164 @@ import java.sql.PreparedStatement
 import java.time.Instant
 import java.util.Date
 
-import anorm.ParameterMetaData.ByteArrayParameterMetaData
-import anorm.{
-  Column,
-  MetaDataItem,
-  ParameterMetaData,
-  RowParser,
-  SqlMappingError,
-  SqlParser,
-  ToStatement
-}
+import anorm._
 import com.daml.ledger.participant.state.v1.Offset
 import com.daml.lf.crypto.Hash
 import com.daml.lf.data.Ref
 import com.daml.lf.value.Value
 
 object Conversions {
 
-  private def stringColumnToX[X](f: String => Either[String, X])(c: Column[String]): Column[X] =
+  private def stringColumnToX[X](f: String => Either[String, X]): Column[X] =
     Column.nonNull((value: Any, meta) =>
-      c(value, meta).toEither.flatMap(x => f(x).left.map(SqlMappingError)))
+      Column.columnToString(value, meta).toEither.flatMap(x => f(x).left.map(SqlMappingError)))
+
+  private final class SubStringToStatement[S <: String] extends ToStatement[S] {
```

Comment:
Am I reading this weird, or does "SubString" in this context mean "something that extends `String`"? If so, that's an overload my brain was not prepared for.

If I am right, can we change this to `SubTypeOfString` or something to make it clearer?

Reply:
I was just using what was there before but sure.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5706/commits/d14449de39fc87d40ac96f7e69f906a20fc0157d

Reply:
Yeah, I know, but it seems like a good opportunity.

Reply:
Thanks!